samples/browser/src/geckoBeta/java/org/mozilla/samples/browser/Components.kt

```diff
@@ -14,10 +15,12 @@ import org.mozilla.samples.browser.request.SampleRequestInterceptor
  * Helper class for lazily instantiating components needed by the application.
  */
 class Components(applicationContext: Context) : DefaultComponents(applicationContext) {
-    override val engine: Engine by lazy {
-        val defaultSettings = DefaultSettings().apply {
-            requestInterceptor = SampleRequestInterceptor(applicationContext)
-        }
-        GeckoEngine(applicationContext, defaultSettings)
-    }
+    // TODO uncomment this; testing this with the SystemEngine because it has a more complete API.
+//    override val engine: Engine by lazy {
+//        val defaultSettings = DefaultSettings().apply {
+//            requestInterceptor = SampleRequestInterceptor(applicationContext)
+//            historyTrackingDelegate = PlacesHistoryTrackingDelegate(applicationContext)
+//        }
+//        GeckoEngine(applicationContext, defaultSettings)
+//    }
```

Comment:
FYI: In the bottom left corner in Android Studio there should be a "build variant" side bar. There you should be able to switch the build variant of `samples-browser`. In this case you can just use `systemUniversalDebug`.